test/unit/db/repos/blocks.js

```diff
@@ -576,7 +576,7 @@ describe('db', () => {
 		});
 
 		describe('getCommonBlock()', () => {
-			it('should call a function with SQL parameters', function*() {
+			it('should call a normal SQL file with parameters if previousBlock is not provided', function*() {
```

Comment:
to follow the used patterns:
``
describe('when previousBlock = undefined')
   it('should call a "getCommonBlock" SQL file with passed parameters')
describe('when previousBlock = 12345')
   it('should call a "getCommonBlockByPreviousBlock" SQL file with passed parameters'
``